### PR TITLE
Rebuild packages for missing metadata vol. 7

### PR DESCRIPTION
**Pull request by Chocimier**
All packages was build (when available) for x86_64, x86_64-musl, armv7hf and aarch64-musl and passed check stage on x86_64 and x86_64-musl. Nothing was tested manually whether it still works.

Homepage is down for chrpath to xsane, so sources are get from elsewhere. Homepage fields itself are not changed.

203 more packages remain to rebuild.

### CODE REVIEW

File: srcpkgs/thunar-volman/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Template file for 'thunar-volman'.
+# Template file for 'thunar-volman'
 pkgname=thunar-volman
 version=0.8.1
-revision=2
+revision=3
 build_style=gnu-configure
 hostmakedepends="pkg-config intltool"
 makedepends="libgudev-devel libnotify-devel libxfce4ui-devel exo-devel"
```
Comment (maxice8) [inline srcpkgs/thunar-volman/template]: GPL-2.0-or-later

File: srcpkgs/tpacpi-bat/template [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 pkgname=tpacpi-bat
 version=3.1
-revision=1
-depends="acpi_call-dkms perl"
+revision=2
 only_for_archs="i686 x86_64 x86_64-musl"
+depends="acpi_call-dkms perl"
 short_desc="ThinkPad ACPI Battery Util using acpi_call"
 maintainer="Daniel A. Maierhofer <git@damadmai.at>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/tpacpi-bat/template]: GPL-3.0-or-later

File: srcpkgs/v4l2grab/template [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # Template file for 'v4l2grab'
 pkgname=v4l2grab
 version=0.4.1
-revision=3
+revision=4
 makedepends="libjpeg-turbo-devel v4l-utils-devel"
 short_desc="Utility for grabbing JPEGs from V4L2 devices"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/v4l2grab/template]: GPL-2.0-or-later

**Comment by Chocimier** [comment]
Changed

**Comment by maxice8** [comment]
merged

**Pull request closed without merging**